support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-id.html

```diff
@@ -0,0 +1,6 @@
+<p>
```

Comment:
I followed the rule in all other inlined help files for fields (the general help files use `<div>` instead).